daemon/api.go

```diff
@@ -2241,6 +2241,9 @@ func postDebug(c *Command, r *http.Request, user *auth.UserState) Response {
 		st.Lock()
 		defer st.Unlock()
 		ensureStateSoon(st)
+	case "get-base-declaration":
+		blob := asserts.Encode(asserts.BuiltinBaseDeclaration())
+		return SyncResponse(blob, nil)
```

Comment:
don't know if you saw my comment on IRC; but I had forgotten about Client.Debug.... so using a SyncResponse is probably ok but we might some labeling/wrapping

map[string]interface{}{"base-declaration": blob}